src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/AssigningSymbolAndItsMemberInSameStatement.cs

```diff
@@ -80,8 +80,8 @@ private void AnalyzeAssignment(OperationAnalysisContext context)
             {
                 assignmentOperation = (ISimpleAssignmentOperation)assignmentOperation.Value;
 
-                var operationValue = assignmentOperation.Target as T;
```

Comment:
Time to start dogfooding C# 8 nullable references in the repo :-)

Any takers @dotnet/roslyn-analysis? Otherwise, I will try to get to it next sprint.

Reply:
We had a similar NRE just a few weeks ago: https://github.com/dotnet/roslyn-analyzers/pull/2762/files

Reply:
Is it just a matter of bumping roslyn version? Or are you actually proposing a full scale NRT roll out in this repo? :)

---
In reply to: [330693127](https://github.com/dotnet/roslyn-analyzers/pull/2894#discussion_r330693127) [](ancestors = 330693127)

Reply:
We can do staged work if that is easier :) Planning to bump master branch to MS.CA 3.0 soon.

Reply:
I suspect we might need to annotate the analyzer/codefix API's first in Roslyn, to have a cleaner annotation here.